modules/contact/src/problems/AugmentedLagrangianContactProblem.C

```diff
@@ -145,6 +165,24 @@ AugmentedLagrangianContactProblem::checkNonlinearConvergence(std::string & msg,
           }
       }
 
+      // next loop over penalty mortar user objects
+      std::vector<MortarUserObject *> mortar_uos;
+      std::list<PenaltyMortarAugmentedLagrangeInterface *> pmuos;
```

Comment:
Does this assume that all penalty friction mortar user objects will be AL?

Reply:
No, only the ones that inherit form the interface